src/Yoti.Auth/DocScan/DocScanService.cs

```diff
@@ -238,5 +302,7 @@ private static string MediaEndpoint(string sessionId, string mediaId)
         {
             return $"sessions/{sessionId}/media/{mediaId}/content";
         }
+
+        private static JsonSerializerSettings DefaultJsonSettings => new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore };
```

Comment:
Nice, could we call this `YotiDefaultJsonSettings` or something? When I saw it declared above I didn't realise it was internally defined

Reply:
Sure. Will amend right now for you.